1-the-basics/css/styles.css

```diff
@@ -0,0 +1,70 @@
+header{
```

Comment:
```suggestion
header {
```
and use classnames over tag names to not force your HTML to match your CSS